AirCasting/SessionViews/ShareViewModal.swift

```diff
@@ -49,26 +57,35 @@ struct ShareViewModal: View {
     
     private var shareButton: some View {
         Button("Share link") {
-            presentationMode.wrappedValue.dismiss()
+            sheet.toggle()
         }.buttonStyle(BlueButtonStyle())
-        .padding(.bottom)
+            .padding(.bottom)
     }
     
     private var descriptionMail: some View {
         Text(Strings.sessionShare.emailDescription)
-            .font(.subheadline)
+            .font(Font.muli(size: 12))
             .foregroundColor(.aircastingGray)
     }
     
     private var oKButton: some View {
         Button("Share file") {
-            presentationMode.wrappedValue.dismiss()
+            if MFMailComposeViewController.canSendMail() {
+                isShowingMailView.toggle()
+            } else {
+                showingAlert = !isShowingMailView
+                print("no email app")
+            }
         }.buttonStyle(BlueButtonStyle())
+            .sheet(isPresented: $isShowingMailView) { MailView(isShowing: $isShowingMailView, result: $mailSendingResult) }
+            .alert(isPresented: $showingAlert) {
+                Alert(title: Text("No Email app"), message: Text("Please, install Apple Email app"), dismissButton: .default(Text("Got it!")))
```

Comment:
don't we have a `Strings` struct to hold all strings? (applies to all user-facing strings in this PR)